### ISSUE TITLE
How to handle series name / number?

### ISSUE THREAD

**Issue by alanorth** [AgriculturalSemantics/cg-core#30]
CGSpace is wondering how to handle series name and number. We currently use something of a convention like "series name; series number" for `dc.relation.ispartofseries`. For example: 

- ELKS Technical Bulletin;2
- Livestock Exchange Think Piece;22
- IFAD Research Series;35
- Annual Report;2006.2
- ILRI Policy Brief;
- Spore;89

Currently the draft CG Core v2 suggests using `cg.series`. Do we think it is important to keep series name and number together or should we use a separate field?

**Comment by marieALaporte** [comment]
if it is used, I propose that we add cg.seriesName et rename cg.serie to cg.seriesNumber. Would that solution work for you?

**Comment by marieALaporte** [comment]
what about you @olatunike, would that work for your system?

**Comment by olatunike** [comment]
Yes, it can still work for me. I'll modify the schema to reflect the modification if that's what we opt for.

**Comment by alanorth** [comment]
Thanks for the responses! On CGSpace we are planning to do the following:

- `dcterms.isPartOf` for series name (see [dcterms guidance](https://www.dublincore.org/specifications/dublin-core/dcmi-terms/#http://purl.org/dc/terms/isPartOf))
- `cg.number` for series number¹ (currently indicated for issue number)

Our preference is to always use a DC field where possible, then a CG Core one, then a CGSpace custom one.

<hr />

¹ I propose to amend the description of `cg.number` to add "series":

> The issue number of a journal, magazine, _series_ or tech-report

**Comment by alanorth** [comment]
I posted an update to my message above based on some testing in DSpace. I had suggested using `cg.number` for both issue number and series number, but that is problematic in DSpace because we have two separate fields for those currently and DSpace gets confused.

@marieALaporte May I suggest that we use `cg.issue` for journal issue, which would be in the same line of thinking as using `cg.volume` anyways. Then we could use `cg.number` or something else for series number.

This is one area where DC can't help us at all because they are really generic and our metadata is very academic/publishing oriented. We have to forge our own path.

**Comment by ayabowork** [comment]
I think these are good suggestions on how CG Core can be improved, so maybe to create the next version. I assume the main reason why we have CG Core is because DC does not have it all.
